cfy_manager/utils/systemd.py

```diff
@@ -59,8 +59,9 @@ def configure(self, service_name, render=True, tmpfiles=False):
 
         logger.debug('Deploying systemd EnvironmentFile...')
         deploy(env_src, env_dst, render=render)
-        logger.debug('Deploying systemd .service file...')
-        deploy(srv_src, srv_dst, render=render)
+        if exists(srv_src):
```

Comment:
Wait, why would that be necessary, if the unit file lives in the RPM?

Reply:
Is it because of rendering? If so, then why isn't that the _only_ way?
And can't we limit rendering to the environment file, and just use bash vars in the unit file?

Reply:
that's the point, this part isn't necessary if the unit file lives in the RPM. So if it's not in `resources/`, then we don't deploy it.
Other usages (units other that restservice) still use this, so this still must be here.

Reply:
Ah, ok, I didn't think about other components. Carry on, then